src/main/java/com/pappaspojkars/pappaspojkarstipsserveradminquizservice/model/Team.java

```diff
@@ -1,29 +1,25 @@
 package com.pappaspojkars.pappaspojkarstipsserveradminquizservice.model;
-
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
-
 import javax.persistence.*;
 import java.util.List;
 
-@AllArgsConstructor
-@NoArgsConstructor
-@Getter
-@Setter
+
 @Entity
 public class Team {
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Integer id;
     private String name;
     private String flag;
-    @ManyToMany
-    private List<Match> match;
+    private Integer refTeam;
```

Comment:
should we make this a Team instead of Integer?

Reply:
Yes. Fixed